### DISCUSSION TITLE
Incorrect display on multi-value variables (SingleStat, Display name, with repeat)

### DISCUSSION THREAD

**Discussion by deka** [grafana/grafana#38386] [Dashboards, Panels & Visualizations]
**What happened**:
With multi-value variable like __value, __text. 
When i use variable in SingleStat with repeat, label "Display name", 
![image](https://user-images.githubusercontent.com/349156/84034475-acf4bf00-a99a-11ea-860c-ac3bb8afb9e5.png)
Not human readable :bowtie:

**What you expected to happen**:
i expect to see "__text", and not "__value"
I suppose always multi-value variable is used on label , we need to see "__text"

**How to reproduce it (as minimally and precisely as possible)**:
- Create a query-based multi-value variable with a simple query as follows:
select id as __value, name as __text from host named __apis__

- Add SingleStat with a query like this : 
![image](https://user-images.githubusercontent.com/349156/84033825-c2b5b480-a999-11ea-89b5-a53c8686caf1.png)
   - "api" field result is in "apis" variable 

- Set Display name (Field / Standard options)
![image](https://user-images.githubusercontent.com/349156/84034124-31930d80-a99a-11ea-8ae0-9a3a71d37cad.png)

- Use repeat over "apis" values
![image](https://user-images.githubusercontent.com/349156/84034242-5a1b0780-a99a-11ea-826b-c10ad7bea9e1.png)

**Anything else we need to know?**:
- __Panel title work as expect (text displayed)__
- I think this issue is related to #25036

**Environment**:
- Grafana version: 7.0
- Data source type & version: MSSQL (multi-value variables)  and Elasticsearch (SingleStat data source)
- OS Grafana is installed on: Windows
- User OS & Browser: Windows/Chrome
- Grafana plugins: -
- Others: -

**Comment by deka** [comment]
@torkelo, you say it's a feature request, Isn't a bug ?

**Comment by torkelo** [comment]
It's currently not easy to fix / support this. More than just a small bug fix

**Comment by marefr** [comment]
@peterholmberg why have this been added to backend platform backlog?

**Comment by deka** [comment]
News ?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
